src/Akka.Persistence.Sql/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -180,7 +180,7 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                 .IsNotColumn();
 
             // We can skip writing tags the old way by ignoring the column in mapping.
-            if (tableConfig.TagWriteMode == TagWriteMode.TagTable)
```

Comment:
The original bug was here, configuration were not being read because it was stored in two different location, depending on which journal plugin were used.
Instead of using convoluted code to check which journal are being configured, we just standardized how we read our configuration and make sure that this code always points to the right information.